numpyro/distributions/constraints.py

```diff
@@ -412,10 +424,47 @@ def log_abs_det_jacobian(self, x, y, intermediates=None):
         return np.full(np.shape(x) if self.event_dim == 0 else np.shape(x)[:-1], 0.)
 
 
+class InvCholeskyTransform(Transform):
```

Comment:
@neerajprad any suggestion for a better name? ^^

Reply:
I think this name is only confusing because of the presence of `LowerCholeskyTransform` which probably should have been named `LowerTrilTransform` or something.